contracts/protocol/lendingpool/LendingPoolCollateralManager.sol

```diff
@@ -67,6 +75,141 @@ contract LendingPoolCollateralManager is
     return 0;
   }
 
+  function liquidateIthacaCollateral(
```

Comment:
lets make sure this function can only be called by fundlock

Reply:
Good catch, adding access control here